src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+/**
+ * Custom Exception for Duke related Errors
+ */
+public class DukeException extends Exception {
```

Comment:
Extend from the `RunTimeException`. 

`Exception` is more broad and can leave out the granularities. Always remember to catch the exception with the smallest scope.